browser/brave_vpn/win/brave_vpn_wireguard_service/status_tray/status_tray_runner.cc

```diff
@@ -239,6 +237,31 @@ void StatusTrayRunner::SubscribeForStorageUpdates() {
       &StatusTrayRunner::OnStorageUpdated, weak_factory_.GetWeakPtr()));
 }
 
+base::FilePath StatusTrayRunner::FindPakFilePath(base::FilePath assets_path,
+                                                 const std::string& locale) {
+  auto pak_path = GetResourcesPakFilePath(assets_path.DirName(), locale);
+  if (base::PathExists(pak_path)) {
+    return pak_path;
+  }
+  pak_path = GetResourcesPakFilePath(assets_path, locale);
+  if (base::PathExists(pak_path)) {
+    return pak_path;
+  }
+  if (locale != "en-US") {
+    return FindPakFilePath(assets_path, "en-US");
+  }
+  NOTREACHED_NORETURN();
```

Comment:
Why not simply using `en-US` here?
```suggestion
  return FindPakFilePath(assets_path, "en-US");
```

Reply:
This will be infinite loop if no file found